lib/Tests/BluetoothActionTests/WriteCharacteristicTests.swift

```diff
@@ -20,22 +20,22 @@ struct WriteCharacteristicRequestTests {
         let characteristicUuid = UUID(n: 2)
         let instance: NSNumber = 3
         let value: Data = Data("4".utf8)
-        let withResponse: Bool = true
+        let withResponse: NSNumber = 1 // true
         let body: [String: JsType] = [
             "device": .string(deviceUuid.uuidString),
             "service": .string(serviceUuid.uuidString),
             "characteristic": .string(characteristicUuid.uuidString),
             "instance": .number(instance),
             "value": .string(value.base64EncodedString()),
-            "withResponse": .number(withResponse ? 1 : 0),
+            "withResponse": .number(withResponse),
```

Comment:
Swift compiler upgrade started flagging the arithmetic-if here as "never reached".